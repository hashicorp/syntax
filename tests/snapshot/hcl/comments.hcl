# Copyright (c) HashiCorp, Inc.

# line comment

// line comment

/*
  Block comment
*/
