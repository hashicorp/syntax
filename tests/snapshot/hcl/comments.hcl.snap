># line comment
#^ source.hcl comment.line.number-sign.hcl punctuation.definition.comment.hcl
# ^^^^^^^^^^^^^ source.hcl comment.line.number-sign.hcl
>
>// line comment
#^^ source.hcl comment.line.double-slash.hcl punctuation.definition.comment.hcl
#  ^^^^^^^^^^^^^ source.hcl comment.line.double-slash.hcl
>
>/*
#^^ source.hcl comment.block.hcl punctuation.definition.comment.hcl
>  Block comment
#^^^^^^^^^^^^^^^^ source.hcl comment.block.hcl
>*/
#^^ source.hcl comment.block.hcl punctuation.definition.comment.hcl
>